Traceback (most recent call last):
  File "c:\users\yamta\miniconda3\lib\site-packages\jupyter_cache\executors\utils.py", line 56, in single_nb_execution
    record_timing=False,
  File "c:\users\yamta\miniconda3\lib\site-packages\nbclient\client.py", line 1077, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "c:\users\yamta\miniconda3\lib\site-packages\nbclient\util.py", line 74, in wrapped
    return just_run(coro(*args, **kwargs))
  File "c:\users\yamta\miniconda3\lib\site-packages\nbclient\util.py", line 53, in just_run
    return loop.run_until_complete(coro)
  File "c:\users\yamta\miniconda3\lib\asyncio\base_events.py", line 583, in run_until_complete
    return future.result()
  File "c:\users\yamta\miniconda3\lib\site-packages\nbclient\client.py", line 527, in async_execute
    cell, index, execution_count=self.code_cells_executed + 1
  File "c:\users\yamta\miniconda3\lib\site-packages\nbclient\client.py", line 822, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "c:\users\yamta\miniconda3\lib\site-packages\nbclient\client.py", line 730, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply['content'])
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
using MAT

# datasets from http://www.rctn.org/bruno/sparsenet/
mat_images = matopen("_static/datasets/IMAGES.mat")
imgs = read(mat_images, "IMAGES")

close(mat_images)
------------------

File "_static/datasets/IMAGES.mat" does not exist and create was not specified

Stacktrace:
 [1] error(::String) at .\error.jl:33
 [2] matopen(::String, ::Bool, ::Bool, ::Bool, ::Bool, ::Bool, ::Bool) at C:\Users\yamta\.julia\packages\MAT\2LFMT\src\MAT.jl:44
 [3] matopen(::String, ::String; compress::Bool) at C:\Users\yamta\.julia\packages\MAT\2LFMT\src\MAT.jl:88
 [4] matopen at C:\Users\yamta\.julia\packages\MAT\2LFMT\src\MAT.jl:88 [inlined]
 [5] #matopen#2 at C:\Users\yamta\.julia\packages\MAT\2LFMT\src\MAT.jl:97 [inlined]
 [6] matopen(::String) at C:\Users\yamta\.julia\packages\MAT\2LFMT\src\MAT.jl:97
 [7] top-level scope at In[1]:4
 [8] include_string(::Function, ::Module, ::String, ::String) at .\loading.jl:1091
LoadError: File "_static/datasets/IMAGES.mat" does not exist and create was not specified

